velox/exec/ExchangeSource.h

```diff
@@ -96,6 +103,14 @@ class ExchangeSource : public std::enable_shared_from_this<ExchangeSource> {
   // ExchangeClient::kBackgroundCpuTimeMs.
   virtual folly::F14FastMap<std::string, int64_t> stats() const = 0;
 
+  /// Returns runtime statistics. ExchangeSource is expected to report
+  /// Specify units of individual counters in ExchangeSource.
+  /// for an example: 'totalBytes ：count: 9, sum: 11.17GB, max: 1.39GB,
+  /// min:  1.16GB'
+  virtual folly::F14FastMap<std::string, RuntimeMetric> metrics() const {
```

Comment:
I think we need to add default implementation to deprecated stats() API that throws VELOX_UNREACHABLE(). This would allow to remove stats() method from ExchangeSource implementations that implement the new API. One all sources are updated we can remove the API altogether.

Reply:
thanks for review this pr. Can you help me review the updated code again.